charts/zitadel/templates/_helpers.tpl

```diff
@@ -36,7 +36,7 @@ Common labels
 {{- define "zitadel.labels" -}}
 helm.sh/chart: {{ include "zitadel.chart" . }}
 {{ include "zitadel.selectorLabels" . }}
-app.kubernetes.io/version: {{ default .Values.image.tag .Chart.AppVersion | quote }}
+app.kubernetes.io/version: {{ default .Chart.AppVersion .Values.image.tag | quote }}
```

Comment:
That's a good one :smile: thanks for fixing this :pray: 

Let's try to make it a bit more intuitive [by using a pipe](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function)?

```suggestion
app.kubernetes.io/version: {{ .Values.image.tag | default .Chart.AppVersion | quote }}
```

Reply:
I take the freedom to commit this :-)